### DISCUSSION TITLE
Ordered Lists in WCAG Guidelines format

### DISCUSSION THREAD

**Discussion by jonoSeniorAccessibilitySpecialist** [w3c/wcag#3410] [Q&A]
We have a query that we have not seen raised before, one of our team has identified that when two decimal points are used in an ordered list that some assistive software will read this as a date.

This happened regardless of the last number having two digits or not.

We feel a solution would be to use an ordered list with a combined Numerical and Alphabetical order to prevent mistaken date.

This is relevant to the success criteria in the WCAG Guidelines where the format switches 1.4.13 Content on Hover Or Focus. This can be read as 1st April 2013. Or in US terms 4th January 2013

Has anyone else had this feedback?

Thanks

**Comment by awkawk** [comment]
Can you provide an example?

**Reply by awkawk** [reply]
Oh, I see. You mean when text like "1.4.13" is found on a page that it can be understood as a date. Yes, it can, but the user can read the characters when what is read involves a date and isn't making sense. Some assumptions by AT are not always correct.

**Reply by jonoSeniorAccessibilitySpecialist** [reply]
Agreed AT may make assumptions, but there is an alternative of using Numbered, Alphabetical Ordered list to mitigate the AT assumptions.
